packages/analytics/analytics-utilities/src/types/explore-v4.ts

```diff
@@ -52,6 +72,15 @@ export const basicExploreAggregations = [
 
 export type BasicExploreAggregations = typeof basicExploreAggregations[number]
 
+export const aiExploreAggregations = [
+  'total_tokens',
+  'request_tokens',
+  'response_tokens',
+  'ai_request_count',
+]
```

Comment:
```suggestion
] as const
```

(Without this, Typescript can't infer things properly.)